templates/default/kafka.service.erb

```diff
@@ -0,0 +1,9 @@
+[Unit]
+After=consul-template.service
+Wants=consul-template.service
```

Comment:
This is too specific for a general purpose cookbook. Suggest to remove these lines, we can add them in our project via local.conf.